src/bindings/config.ts

```diff
@@ -32,8 +32,8 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
 
   const botConfig: BotConfig = {
     log: {
+      logEnvironment: process.env.LOG_ENVIRONMENT || "production",
```

Comment:
What does `logEnvironment` do?

Reply:
If its development.. It logs on the console for easy log tracking.. Would come in handy, I've always wanted to see logs without going to memzo dashboard or using console.log and the forgetting to remove it